sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/GridUtils.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.portal.sampletracking.components
+
+/**
+ * <h1><short description></h1>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <versiontag>
+ *
+*/
```

Comment:
I think adding a description would make sense something akin to the offer-manager 
```suggestion
/**
 * A helper class with static utility functions for Vaadin Grids.
 *
 * @since 1.0.0
 */
```

Reply:
We could also think about moving such helper classes targeting vaadin components to a specific repository e.g. core-utils-lib instead of defining them  seperately in each portlet.

Reply:
I agree. We should move the GridUtils functionality to another repo. Maybe we should bring that up in our daily meeting

Reply:
👍